web/packages/teleport/src/Console/DocumentSsh/DocumentSsh.tsx

```diff
@@ -77,29 +78,39 @@ export default function DocumentSsh({ doc, visible }: PropTypes) {
           afterClose={handleCloseFileTransfer}
           backgroundColor={colors.primary.light}
           transferHandlers={{
-            getDownloader: async (location, abortController) =>
-              getHttpFileTransferHandlers().download(
+            getDownloader: async (location, abortController) => {
+              const wanResponse = await auth.getAssertionResponseIfRequired({
+                node: { node_name: doc.clusterId, login: doc.login },
```

Comment:
Setting the node name field to a variable called "cluster ID" seems suspicious - are you sure that's what we want?

Reply:
Good catch on this one. I was hitting the first check of "MFA check always" and so it wasn't getting to the node check on the backend. Changed to `serverId` and updated my local config to reflect this check and working as intended again.